internal/server/async_tcp.go

```diff
@@ -5,20 +5,27 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"io"
 	"net"
+	"strings"
 	"sync"
 	"syscall"
 	"time"
 
+	eval2 "github.com/dicedb/dice/internal/eval"
```

Comment:
`eval2` required due to some name collision?

Reply:
done